test/dest-symlinks.js

```diff
@@ -49,6 +59,8 @@ describe('.dest() with symlinks', function() {
       expect(files.length).toEqual(1);
       expect(file.symlink).toEqual(symlink);
       expect(files[0].symlink).toEqual(symlink);
+      expect(files[0].stat).toExist();
+      expect(files[0].stat.isSymbolicLink()).toBe(true);
```

Comment:
I don't like testing `isSymbolicLink` and would rather just test `isSymbolic` (the public interface)

Reply:
Okay, sure.